.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
```

Comment:
I understand that I haven't uploaded a .gitignore and that can be frustrating.
However, I prefer to keep PR clear from unrelated changes.